### ISSUE TITLE
Maven coordinates changed

### ISSUE THREAD

**Issue by eliasbalasis** [Microsoft/elastic-db-tools-for-java#13]
I believe the Maven coordinates have changed from
<dependency>
      <groupId>com.microsoft.azure</groupId>
      <artifactId>elastic-db-tools</artifactId>
      <version>1.0.0</version>
</dependency>
to
<dependency>
    <groupId>com.microsoft.azure</groupId>
    <artifactId>azure-elasticdb-tools</artifactId>
    <version>1.0.0</version>
</dependency>
is this true?
If yes, can you please adjust the code and pages?

**Comment by Blackbaud-MattWalsh** [comment]
The versioning is all kinds of messed up.

According to the master branch, the source code is still on 1.0.1, but it's not published to Maven central.

The latest branch in Maven central is 1.1.0, but doesn't exist as a tag in the GitHub repo.

When trying to use version 1.1.0, I'm getting an error that doesn't parse connection string properties correctly, but works fine with version 1.0.1 (the unpublished current version).

**Comment by eliasbalasis** [comment]
I fear there is a lot more to this.

I found similar problems when passing SQL Server port to shard definitions as per samples, but it worked without a port (the default port is used instead)

Also, the shard map manager and shard database scripts cannot be dicsovered at runtime and as a result sharding structure preparation fails.
There is change between 1.0.0 and 1.1.0 in this area trying to dynamically discover the script files in executable JAR at runtime but fails to discover the scripts.
The only solution I could find is change the code and embed the scripts inside a custom clone of elastic-db-for-java JAR adjusting "Scripts" class.

I hope this helps.

**Comment by Blackbaud-MattWalsh** [comment]
I found the same thing. There's too much hand waving to get this to work. I resorted to forking just the lib (not samples/) and dealing with that directly. At least there's options so you're not stuck :/

**Comment by eliasbalasis** [comment]
Indeed, it needs tweaking and it has been hard to make it work but I managed to make it work for my team.
In our implementation we are getting the JDBC connection from elastic-db-tools and matching it to a JPA entity manager.
